### DISCUSSION TITLE
Adding date / time selection range for events

### DISCUSSION THREAD

**Discussion by jhallenb** [blakeblackshear/frigate#1366] [Ideas]
Hi,
How difficult would it be to add date and time range selectors for the events?
Now all events are listed, and of course one could use find on the page to find a certain date at least but it would be a big improvement if one could filter on dates and times with dropdowns or sliders.

Not familiar with the coding of this web gui but if some one could add this feature I believe the overall gui would be much more useful.

**Comment by PrplHaz4** [comment]
Others agree - additional discussion here: https://github.com/blakeblackshear/frigate/issues/1318

**Reply by muzzak123** [reply]
I agree too.  Unfortunately that discussion #1318  has now been closed by the stale bot. It is hard to scroll through a lot of events looking for a certain time or day.
